### PR TITLE
Update README.MD - add more links for compatibility

### PR DESCRIPTION
**Pull request by PlayfulPiano**
No links were listed for a couple mod version incompatibilities. they're now listed.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -33,10 +33,10 @@
 * OptiFine [Disabled, won't fix]
 * FastCraft [Disabled, won't fix]
 * BetterFPS [Disabled, won't fix]
-* Amazing Trophies 1.2.1 and below - use 1.2.2+
-* LWJGL3ify 1.5.4 and below - Use the latest version available, 1.5.7+
-* ArchaicFix 0.6.2 and below - Use 0.7.0 or above
-* Hodgepodge 2.4.3 and below - [Use 2.4.4 or above](https://github.com/GTNewHorizons/Hodgepodge/releases)
+* [Amazing Trophies 1.2.1 and below](https://github.com/GTNewHorizons/Amazing-Trophies/releases) - use 1.2.2+
+* [LWJGL3ify 1.5.4 and below](https://github.com/GTNewHorizons/lwjgl3ify/releases/) - Use the latest version available, 1.5.7+
+* [ArchaicFix 0.6.2 and below](https://www.curseforge.com/minecraft/mc-mods/archaicfix/files/all?page=1&pageSize=20) - Use 0.7.0 or above
```
Comment (Alexdoru) [inline README.MD:38]: I think archaix fix is bundled within angelica so it shouldn't say `Use 0.7.0 or above` but rather that it comes with angelica

**Review by Alexdoru — CHANGES_REQUESTED** [review]

File: README.MD (hunk 1 above)
Comment (PlayfulPiano) [reply README.MD:38]: You sure? cuz uh, I have archaicfix manually installed in my modlist separate from angelica lol

Comment (Alexdoru) [reply README.MD:38]: I'm not sure, but there is a bunch of stuff named "archaic" in the angelica source as well as a "archaicfix" access transformer, so idk would need to ask @mitchej123

Comment (Cleptomania) [reply README.MD:38]: It is not included in Angelica, there’s a small amount of things that we ripped from it

**Review by Alexdoru — APPROVED** [review]

**Pull request merged**